libs/utils/src/lib/models/gatewayServiceConfig.ts

```diff
@@ -0,0 +1,3 @@
+export interface GatewayServiceConfig {
```

Comment:
Let's leave this in the services package instead of the utils package, just have it below the `IGatewayService` interface in `libs/services/designer-client-services/src/lib/gateway.ts`